index.js

```diff
@@ -21,7 +21,7 @@ express()
   .get('/db', async (req, res) => {
     try {
       const client = await pool.connect();
-      const result = await client.query('SELECT * FROM test_table');
+      const result = await client.query(`SELECT * FROM products where prod_name ='${req.query.searchParam}'`);
```

Comment:
This query works but if you would like to make it better, I would suggest using LIKE plus a wildcard character instead of =.  For example: WHERE prod_name LIKE 'banana%'
This example would return anything starting with banana with 0 or more characters after.  For example, it would return both banana and banana bread if those items were in the table.
I'm not 100% sure how this works with a pass through variable but my guess would be that you would have to write
SELECT * FROM products where prod_name ='${req.query.searchParam}%'

Reply:
Thank you! Appreciate your suggestion. I tried to change the query however it stopped displaying the product name. I will look into this more and try to have it working for the next sprint!